server/middlewares/fetch-user.js

```diff
@@ -169,32 +189,30 @@ module.exports = function fetchUser(req, res, next) {
 
   return userPromise
     .then((payload = {}) => {
-      const segments = _.map(payload.segments, s =>
-        _.pick(s, "id", "name", "type", "updated_at", "created_at"));
-      const randKeys = _.sampleSize(_.omit(_.keys(payload.user), "account"), 3);
+      const segments = _.map(payload.segments, formatSegment);
+      const account_segments = _.map(payload.account_segments, formatSegment);
```

Comment:
Personally I would use lowerCamelCase for all const names, and loose a little bit of shortcut at the end.